backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -48,6 +48,10 @@
             <%= f.text_field :shipments_number_start %>
           </div>
 
+          <div class="field">
+            <%= label_tag :q_shipment_state, t('spree.shipment_state') %>
+            <%= f.select :shipment_state_eq, %i[backorder canceled partial pending ready shipped].map { |s| [t("spree.shipment_states.#{s}"), s] }, { include_blank: true }, { class: "custom-select fullwidth" } %>
```

Comment:
I prefer to have a longer line (especially because it's already very long) but without the single-letter variable. Can we please change `s` into `state`?